### ISSUE TITLE
Show loader while using lazyloading and limitSelection together?

### ISSUE THREAD

**Issue by charumaheshwari** [CuppaLabs/angular2-multiselect-dropdown#354]
I am using following settings to show angular multidropdown
  ```
public dropdownSettings = {
    text: 'Separate addresses with commas',
    classes: 'teacher-select',
    primaryKey: 'uuid',
    labelKey: 'email',
    badgeShowLimit: 1,
    limitSelection: 10,
    lazyLoading: true
  };
```
However, in UI "loader get displayed at the right side of Select All HTML element.  Assuming "limitSelection" hides "Select All" functionality from dropdown.

![image](https://user-images.githubusercontent.com/2443298/53225529-5a589c00-369e-11e9-9ee4-ffb378bdd40a.png)

Issue is  where/how to show loader in UI while using "limitSelection".
Kindly advice if I am missing anything while configuring the settings.

**Comment by CuppaLabs** [comment]
@charumaheshwari ETA 1 day .

**Comment by pratik-joshi-cactus** [comment]
@CuppaLabs  did you fix  it(as eta was 1 day)?  Thanks

**Comment by CuppaLabs** [comment]
fixed. fix available in v4.6.0
